debian/control

```diff
@@ -12,9 +12,9 @@ Build-Depends:
  python-numpy,
  python-wheel,
  python-setuptools,
- curl,
+ curl, libflatbuffers-dev, flatbuffers-compiler
```

Comment:
tensorflow-lite-dev depends should also include `libflatbuffers-dev`

Reply:
Also, this needs to be updated for tizen packaging?

Reply:
tizen packaging (https://review.tizen.org/gerrit/#/admin/projects/platform/upstream/tensorflow) have dependency to flatbuffers-dev.
One problem is that Tizen and Debian packaging are managed in different repos.
Tizen provides tf-lite only and debian provides both tf and tf-lite.
What is a good policy for managing repo for tensorflow?

Reply:
I think all changes and maintaining should be performed in `review.tizen.org`.
And this repo should just sync with review.tizen.org

Reply:
Tizen packaging only provides tflite.
Doesnt meant that tizen repo cannot maintain debian packaging.